### DISCUSSION TITLE
Does it matter if this project is in a Python or Javascript backend framework?

### DISCUSSION THREAD

**Discussion by thakopian** [floss-and-code/stripe-hackathon#9] [Q&A] [answered]
This is just a general question. I understand the project today will be in JS stack but curious if it would make any difference to use a different stack with this repo should we fork it and try it out. Any hints, documents or related repos with Python stack would be great.

**Comment by Gattalraouf** [comment, marked as answer]
Hello @thakopian ,
The challenge here is to improve the existing [nextjs-subscription-payments](https://github.com/thorwebdev/nextjs-subscription-payments) template (after forking it of course), using one of the ways explained in [here](https://github.com/floss-and-code/stripe-hackathon). Based on that It's recommended to keep using the same stack as in the template. Yet it's a fact that improving the template, can include migrating the CSS to a python project for example.

Just bear in mind that in addition to the mentioned template, all samples [here](https://github.com/stripe-samples) are valid in the challenge, also that you can build your own template if you are interested in doing so. 

Good luck and don't forget to have fun!!!!

**Reply by thakopian** [reply]
That makes sense. I think starting with the fork then formatting a new template down the road makes sense. Btw will this repo remain open for comments and questions for a while? It would be great to come back to this after the Github event to ask questions about the template via Github.

**Reply by Gattalraouf** [reply]
Yes the repo will remain open. Maybe there won’t be formal comments and questions, yet it’s open source so make sure that it will be open to anyone to continue using the discussions area to ask and/or answer questions even after the Hackathon.
